### ISSUE TITLE
Ubuntu Software is Invisible

### ISSUE THREAD

**Issue by Lucas-Bide** [freefreeno/Materia-Manjaro-Dark#1]
The program opens but doesn't have a background or images. Only the text is visible. This is on Ubuntu 20.04.

**Comment by freefreeno** [comment]
Is it the software store ????

**Comment by Lucas-Bide** [comment]
Yes. I noticed it on another theme as well, but I thought I should mention it in case the Ubuntu upgrade broke a common theme dependency.
